Userland/Libraries/LibJS/Runtime/Temporal/Calendar.cpp

```diff
@@ -718,11 +718,13 @@ i32 iso_year(Object& temporal_object)
     // NOTE: Asserted by the VERIFY_NOT_REACHED at the end
 
     // 2. Return 𝔽(temporalObject.[[ISOYear]]).
-    // TODO: add the rest of the builtins with a [[ISOYear]] slot (PlainYearMonth, PlainMonthDay)
     if (is<PlainDate>(temporal_object))
         return static_cast<PlainDate&>(temporal_object).iso_year();
     if (is<PlainDateTime>(temporal_object))
         return static_cast<PlainDateTime&>(temporal_object).iso_year();
+    if (is<PlainYearMonth>(temporal_object))
+        return static_cast<PlainYearMonth&>(temporal_object).iso_year();
```

Comment:
Same in the `LibJS: Handle ZonedDateTime in ISO{Year,Month,MonthCode,Day}` commit as above